src/components/Comments.js

```diff
@@ -0,0 +1,141 @@
+
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom";
+import { Link, Modal, Tooltip, IconButton, Container, Box, Grid, Card, CardActions, CardContent, CardMedia, Button, Typography, Chip } from '@mui/material';
+import PersonIcon from '@mui/icons-material/Person';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import AddCommentIcon from '@mui/icons-material/AddComment';
+import AddCommentModal from './AddCommentModal';
+import DeleteCommentModal from './DeleteCommentModal';
+import { getComments } from "../api"
+
+const Comments = () => {
+    const [isLoading, setIsLoading] = useState(false)
+    const [comments, setComments] = useState([])
+
+    const [openDelete, setOpenDelete] = useState(false);
+    const [openAdd, setOpenAdd] = useState(false);
+    const [addCommentArticleId, setAddCommentArticleId] = useState(null)
+
+    const handleDeleteOpen = () => setOpenDelete(true);
+    const handleDeleteClose = () => setOpenDelete(false);
```

Comment:
This could be refactored to one function handleDeleteChange which would setDelete to be !delete